src/extensions/surveys/surveys-utils.tsx

```diff
@@ -563,19 +563,41 @@ export const createShadow = (styleSheet: string, surveyId: string, element?: Ele
     return shadow
 }
 
-export const sendSurveyEvent = (
-    responses: Record<string, string | number | string[] | null> = {},
-    survey: Survey,
+type SendSurveyEventArgs = {
+    responses: Record<string, string | number | string[] | null>
+    survey: Survey
     posthog?: PostHog
-) => {
+    isSurveyCompleted: boolean
+    surveySubmissionId: string
+}
+
+export const sendSurveyEvent = ({
+    responses,
+    survey,
+    posthog,
+    isSurveyCompleted,
+    surveySubmissionId,
+}: SendSurveyEventArgs) => {
     if (!posthog) {
         logger.error('[survey sent] event not captured, PostHog instance not found.')
         return
     }
+    if (!isSurveyCompleted && !survey.enable_partial_responses) {
+        return
+    }
     localStorage.setItem(getSurveySeenKey(survey), 'true')
 
+    logger.info('[PostHog Surveys] survey partial response payload', {
```

Comment:
```suggestion
    logger.info('survey partial response payload', {
```

createLogger already has a prefix